test/e2e/generated.svg.test.ts

```diff
@@ -17,9 +17,11 @@ import 'jest-playwright-preset';
 import type { Page } from 'playwright';
 import { BpmnPageSvgTester } from './helpers/visu/bpmn-page-utils';
 
+const diagramsDirectoryName = 'svg';
```

Comment:
❓ To simplify usage, is it possible to rename it in all test files ?

```suggestion
const diagramSubfolder = 'svg';
```

Reply:
Yes! see 81df035